build-support/bin/ci.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+function clone_repo {
+  local name=$1
+  local repo=$2
+  local branch=$3
+  git clone $repo || {
+    echo "Unable to clone $name from $repo"
+    exit 1
+  }
+  cd $name || {
+    echo "Unable to get $name from $repo"
+    exit 2
+  }
+  git checkout $branch || {
+    echo "Unable to checkout branch $branch for $name from $repo"
+    exit 3
+  }
+  cd ..
+}
+
+function publish_local {
+  local name=$1
+  local target=$2
+  cd $name || {
+    echo "Unable to get $name for publish local $target"
+    exit 4
+  }
+  ../../../sbt ++$TRAVIS_SCALA_VERSION $target || {
+    echo "Unable to publish local $name with target $target"
+    exit 5
+  }
+  cd ..
+}
+
+function bootstrap_scrooge {
+  local current_branch=$(git rev-parse --abbrev-ref HEAD)
+  if [ "$current_branch" != "master" ]; then
+    mkdir -p build-support/tmp
```

Comment:
Duh for me !! ... you are right, that's even better (easier and cleaner). I will change it to mktemp instead.